src/modules/tsilo/tsilo.c

```diff
@@ -250,15 +250,25 @@ static int fixup_ts_append(void** param, int param_no)
  */
 static int w_ts_append(struct sip_msg* _msg, char *_table, char *_ruri)
 {
-	str ruri = {0};
+	str tmp  = STR_NULL;
```

Comment:
I think this change should be done into ts_append, not in the wrapper. What can be changed by branch routes is the  ruri that you pass to ts_append, so saving the original one at this level wouldn't solve the issue, if I'm not wrong.